spec/controllers/idv/in_person_controller_spec.rb

```diff
@@ -17,9 +17,9 @@
       expect(subject).to have_actions(
         :before,
         :confirm_two_factor_authenticated,
-        :initialize_flow_state_machine,
-        :ensure_correct_step,
         :set_usps_form_presenter,
+        :redirect_unless_enrollment,
+        :initialize_in_person_session,
```

Comment:
Good to see this added. 💪🏻

Reply:
I was going to ask if there's a benefit to increasing test coverage for the functionality of the`initialize_in_person_session` method, but I realize it's tricky to test private methods.  I'm curious what you think.

Reply:
Good suggestion! I added a test to in_person_controller_spec to check that the session is initialized correctly when the index action is called, which indirectly tests the initialize_in_person_session method 👍